### PR TITLE
added coding for reptils js

### PR DESCRIPTION
**Pull request by Moon2024-light**
please review my reptiles onclick event file thanks

### CODE REVIEW

File: html/reptiles.html [hunk 1]
```diff
@@ -10,50 +11,96 @@
     <link rel="stylesheet" href="../styles/global.css">
     <link rel="stylesheet" href="../styles/variables.css">
     <link rel="stylesheet" href="../styles/reptiles.css">
+    <script src="../scripts/menu.js" defer></script>
+    <script src="../scripts/reptiles.js"defer></script>
```
Comment (TobiasKjernell) [inline html/reptiles.html]: make a space here between "defer" and script name, just for some beauty

**Review by idacarlin95 — APPROVED** [review]
think it looks good

**Review by Moon2024-light — COMMENTED** [review]
fixed js link

**Review by TobiasKjernell — APPROVED** [review]

**Review by TobiasKjernell — CHANGES_REQUESTED** [review]
need to add length/weight, found at etc. some informations is missing

**Review by Moon2024-light — COMMENTED** [review]
added coding please check thanks

**Review by jasminedns — APPROVED** [review]
looks ok!

**Review by TobiasKjernell — APPROVED** [review]

**Pull request merged**